commons/src/connections/events.ts

```diff
@@ -38,7 +38,7 @@ export const fetchEvents = async (
 export const fetchCalendars = async (
   query: Events.CalendarQuery,
 ): Promise<Events.Calendar[]> => {
-  return Promise.all(
+  return Promise.allSettled(
```

Comment:
Why allSettled here? Do we gracefully handled a failed-to-fetch-calendar case? Wouldn't that be perhaps misleading to the end user?

Reply:
With `Promise.all`, the entire request fails and errors out vs with `Promise.allSettled` if a request failed here, we skip it. Do you think that should not be the case? 
I think it makes sense to error out when we have the error component working, so that the user knows why exactly the request failed.